base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1844,13 +1844,13 @@
       </value>
     </entry>
     <entry>
-      <key>Reflectometry Front Panel</key>
+      <key>Reflectometry OPI</key>
```

Comment:
This will break existing saved device screens, can we have a step to fix them in the upgrade script?